check_net.sh

```diff
@@ -13,7 +13,7 @@ do
 	works='true';
 	for check_address in "${ip_addresses[@]}"
 	do
-		if ping -c 1 $check_address > /dev/null;
+		if ping -W 5 -c 1 $check_address > /dev/null;
```

Comment:
BTW you can have a condition if it's BSD/Darwin because linux ping utility is different.
ping(8) timeout declared using -t